docs/_includes/common/filter-button.html

```diff
@@ -6,40 +6,48 @@ <h2><span class="secondary-text">@ebay/skin/</span>filter-button</h2>
     <h4>Default Filter Button</h4>
     <div class="demo">
         <div class="demo__inner">
-            <button type="button" class="filter-button">
-                <span class="filter-button__cell">Button</span>
-            </button>
-            <a href="http://www.ebay.com" class="filter-link">
-                <span class="filter-link__cell">Link</span>
-            </a>
+            <div class="filter-button__wrapper">
```

Comment:
This is invalid BEM. A child class cannot be at the root. What is the purpose of this wrapper? If it's to help with the layout of buttons side by side, then it should be moved to the field module.

Reply:
So like, `field__button-group`?

Reply:
Yeah, and/or `field__filter-group`

Reply:
It's possible we could just keep this within the filter module too. Something like `filter-group`.